### DISCUSSION TITLE
Is it possible to configure renovate to ignore semver versioning and to always update to "@latest" tag

### DISCUSSION THREAD

**Discussion by sattaman** [renovatebot/renovate#8541] [Request Help]
We have an internal package being published by CI but this is not correctly following semver versioning, publishes are being published as 

0.1.0-107

where 107 is a build counter. 

Is it possible to configure renovate to always attempt to use whatever comes back from @latest tag? It is currently ignoring these. 

Thanks

**Comment by rarkins** [comment]
Is it an npm package? If so then configure followTag=latest in a package rule for that package

**Reply by sattaman** [reply]
Great thanks!

**Reply by DerRockWolf** [reply]
Does `followTag` also work for docker images?

**Reply by rarkins** [reply]
No. Please start a new discussion describing your requirements
